src/async.tex

```diff
@@ -259,13 +259,13 @@
   &
 Shall not exit via an exception. \br\textit{Precondition:} A preceding call \tcode{x2.on_work_started()} where \tcode{x1 == x2}.  \\ \rowsep
 
-\tcode{x1.dispatch(std::move(f),a)}  &
+\tcode{x1.dispatch(\br\mbox{\ \ }std::move(f),~a)}  &
   &
 \effects Creates an object \tcode{f1} initialized with \tcode{\textit{DECAY_COPY}(forward<Func>(f))} (\CppXref{thread.decaycopy}) in the current thread of execution . Calls \tcode{f1()} at most once. The executor may block forward progress of the caller until \tcode{f1()} finishes execution. Executor implementations should use the supplied allocator to allocate any memory required to store the function object. Prior to invoking the function object, the executor shall deallocate any memory allocated. \enternote Executors defined in this Technical Specification always use the supplied allocator unless otherwise specified. \exitnote \br
 \sync The invocation of \tcode{dispatch} synchronizes with (\CppXref{intro.multithread}) the invocation of \tcode{f1}.  \\ \rowsep
 
-\tcode{x1.post(std::move(f),a)}\br
-\tcode{x1.defer(std::move(f),a)}  &
+\tcode{x1.post(std::move(f), a)}\br
+\tcode{x1.defer(std::move(f), a)}  &
```

Comment:
In 4a0c325ab87b60710b52fbff08293330291996e9 Lars added NBSP here instead, which doesn't make any difference to the generated PDF. Any reason to prefer one over the other?